infrastructure/docker/services/php-base/Dockerfile

```diff
@@ -13,13 +13,15 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
 
-ARG PHP_VERSION=7.4
+ARG PHP_VERSION
+
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         php${PHP_VERSION}-apcu \
+        php${PHP_VERSION}-bcmath \
+        php${PHP_VERSION}-cli \
         php${PHP_VERSION}-common \
         php${PHP_VERSION}-curl \
-        php${PHP_VERSION}-iconv \
```

Comment:
Why do you remove this packages?

Reply:
Because it failed in some PHP version, but I can remembe which one (probably 7.2)

Reply:
IMHO, it's better to keep it, and people that use 7.2 will have to remove it manually.
https://www.php.net/supported-versions.php => PHP 7.2 is not supported anymore (security fix only until end of 2020). So I don't really care if the docker started does not work out of the box with unsupporte PHP version :)

Reply:
I just tested with this package required and it everything worked fine on PHP versions from 7.2 to 7.4, so let's keep it as it is \o/